src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlDatabaseInstanceFailoverGroup.md

```diff
@@ -0,0 +1,118 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version:
```

Comment:
@jugeorge for each of the markdown files you are adding, please make sure that
- this `online version` section has a corresponding URL to the docs site (the `ValidateHelpIssues.csv` file in the build should tell you what to put in this section`), and
- the examples you add have the corresponding output that the user would see when running the given command(s)

Reply:
@cormacpayne Just wanted some clarification on these points. I was looking in the ValidateHelpIssues.csv file, but it doesn't reference the new cmdlets I've added or list any of the URLs. Do the nugets need to be published for these to get filled?

Also, I was looking for examples in other help files, just to see how the output should be formatted. Could you point me to them?